cmd/build/v2/build.go

```diff
@@ -180,7 +180,6 @@ func (bc *OktetoBuilder) buildService(ctx context.Context, manifest *model.Manif
 }
 
 func (bc *OktetoBuilder) buildSvcFromDockerfile(ctx context.Context, manifest *model.Manifest, svcName string, options *types.BuildOptions) (string, error) {
-	oktetoLog.Information("Building image for service '%s'", svcName)
```

Comment:
I think we should at least log it into `oktetoLog.Info`. That way we can have a better doctor file